Probesys/bileto
# imp: Display only tech users in lists of assignees

## Related issue(s)



https://github.com/Probesys/bileto/issues/273

## Changes



- use the UsersSorter in the ActorsLister
- limit the actors lists to the users of the current organization
- limit the actors lists to the users of the accessible organizations
- allow to list users with either "user", "tech" or both roles

## How to test manually



- open the lists of requesters (new ticket, edit actors and filter actors) → check only users of the current organization are listed
- open the lists of assignees (new ticket, edit actors and filter actors) → check only **tech** users of the current organization are listed

## Checklist
